gdenaro_es1.txt

```diff
@@ -0,0 +1,59 @@
+•  Spiegare i vantaggi di un sistema di versioning all’interno di un progetto. 
+
+	Permette agli utenti di lavorare su un unico progetto in maniera simultanea, grazie alla possibilità di creare un workflow di lavoro in cui le operazioni basilari che si svolgono all’interno di un progetto (patch, aggiunta di nuove features, reset, ecc) risultano chiare e separate. Inoltre è un modo per risparmiare l’utilizzo di dati visto che il file su cui si lavora rimane unico.
+ 
+•  Indicare i comandi che conosci per la creazione di un repository. 
+
+	In locale: dopo aver creato una directory con il comando “ mkdir <options> <path> ” la cartella può essere inizializzata come una repository con il comando  “ git init “. 
+	In remoto: sul sito github tramite il comando new. 
```

Comment:
con git init crei un repo in locale ex novo; puoi scaricare in locale un repo già esistente anche con il comando git clone